docs/data_model/index.md

```diff
@@ -8,19 +8,19 @@ The Data Model, strongly inspired by [CybOX](https://cyboxproject.github.io/), i
 
 |Object|Actions|Fields|
 |---|---|---|
-|**[authentication](authentication)**|`error`<br />`failure`<br />`success`|`ad_domain`<br />`app_name`<br />`auth_service`<br />`auth_target`<br />`decision_reason`<br />`fqdn`<br />`hostname`<br />`fqdn`<br />`method`<br />`response_time`<br />`target_ad_domain`<br />`target_uid`<br />`target_user`<br />`target_user_role`<br />`target_user_type`<br />`uid`<br />`user`<br />`user_agent`<br />`user_role`|
-|**[driver](driver)**|`load`<br />`unload`|`base_address`<br />`fqdn`<br />`hostname`<br />`image_path`<br />`md5_hash`<br />`module_name`<br />`pid`<br />`sha1_hash`<br />`sha256_hash`<br />`signer`<br />`signature_valid`|
-|**[email](email)**|`block`<br />`delete`<br />`deliver`<br />`redirect`<br />`quarantine`|`action_reason`<br />`attachment_mime_type`<br />`attachment_name`<br />`attachment_size`<br />`date`<br />`dest_address`<br />`dest_ip`<br />`dest_port`<br />`from`<br />`message_body`<br />`message_links`<br />`message_type`<br />`return_address`<br />`server_relay`<br />`smtp_uid`<br />`src_address`<br />`src_domain`<br />`src_ip`<br />`src_port`<br />`subject`<br />`to`|
-|**[file](file)**|`acl_modify`<br />`create`<br />`delete`<br />`modify`<br />`read`<br />`timestomp`<br />`write`|`content`<br />`company`<br />`creation_time`<br />`file_name`<br />`file_path`<br />`file_uid`<br />`file_user`<br />`file_extension`<br />`file_gid`<br />`file_gid`<br />`fqdn`<br />`hostname`<br />`image_path`<br />`link_target`<br />`md5_hash`<br />`mime_type`<br />`pid`<br />`ppid`<br />`previous_creation_time`<br />`sha1_hash`<br />`sha256_hash`<br />`signer`<br />`signature_valid`<br />`uid`<br />`user`|
-|**[flow](flow)**|`end`<br />`message`<br />`start`|`application_protocol`<br />`content`<br />`dest_fqdn`<br />`dest_hostname`<br />`dest_ip`<br />`dest_port`<br />`end_time`<br />`exe`<br />`fqdn`<br />`hostname`<br />`image_path`<br />`in_bytes`<br />`network_direction`<br />`out_bytes`<br />`packet_count`<br />`pid`<br />`ppid`<br />`proto_info`<br />`protocol`<br />`src_fqdn`<br />`src_hostname`<br />`src_ip`<br />`src_port`<br />`start_time`<br />`tcp_flags`<br />`transport_protocol`<br />`uid`<br />`user`|
```

Comment:
Double check stuff like this 'protocol' field which is not specified anywhere else.  Should it be?  Or is this location the mistake?